Detectors/FIT/FT0/test/test_raw_conversion_ft0.cxx

```diff
@@ -0,0 +1,73 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include <TSystem.h>
+#include <TTree.h>
+#include <boost/test/tools/old/interface.hpp>
+#define BOOST_TEST_MODULE Test FT0RAWIO
+#define BOOST_TEST_MAIN
+#define BOOST_TEST_DYN_LINK
+#include <boost/test/unit_test.hpp>
+#include "DetectorsCommonDataFormats/NameConf.h"
+#include "Framework/Logger.h"
+#include "DataFormatsFT0/Digit.h"
+#include <TFile.h>
+#include <TRandom.h>
+#include <TStopwatch.h>
+#include <cstring>
+
+using namespace o2::ft0;
+
+BOOST_AUTO_TEST_CASE(RAWTest)
+{
+  gSystem->Exec("$O2_ROOT/bin/o2-sim -n 10 -m FT0 -g pythia8");
+  gSystem->Exec("$O2_ROOT/bin/o2-sim-digitizer-workflow -b");
+  TFile flIn("ft0digits.root");
```

Comment:
Nice that you put down a test.

However, I am not sure if we should add yet another simulation in the CI. We already have the full-system test which is doing most of these steps. Or we have other short simulations that are already running in the CI.

So in order not to waste resources one should use the sim products produced by them. (This should be possible by chaining unit tests).

I know that this may required some careful (frameworked) layout. @davidrohr : Any thoughts? We could for instance run the full-system test and allow detectors to hook in detector-specific tests thereafter?

Reply:
@sawenzel I've asked @AllaMaevskaya to add such a test since we from time to time we observe the differences in generated and decoded data. 
I agree, we can save time by using FST data. I would propose to convert this simple executable, which depending on the provided option does the simulation or starts the check from https://github.com/AliceO2Group/AliceO2/blob/3b7b4aeb8d041606840fa37f868371bfad94ec72/Detectors/FIT/FT0/test/test_raw_conversion_ft0.cxx#L51
Then it can be called from the ``full_system_test.sh``.

@AllaMaevskaya I see you are checking only number of digits and channels. Is not it worth also to check the content of channels/digits, e.g. compare the histos of channel IDs, amplitudes etc.?

Reply:
Comparison of the histos can be added , I just wanted to start from simplest check.